src/ensembl/src/shared/components/instant-download/instant-download-fetch/fetchSequenceChecksums.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { gql } from '@apollo/client';
+
+import { client } from 'src/gql-client';
+
+import { ProductGeneratingContext } from 'src/content/app/entity-viewer/types/productGeneratingContext';
+
+type GeneFragment = {
+  transcript: {
+    product_generating_contexts: ProductGeneratingContextFragment[];
+  };
+};
+
+export type ProductGeneratingContextFragment = Pick<
+  ProductGeneratingContext,
+  'cdna' | 'cds' | 'product'
+>;
```

Comment:
Umm, this doesn't look like the correct type. CDNA, CDS and product requested from Thoas contain just a single field.

Reply:
Created a new type for this

Reply:
✅

Reply:
I wonder if we can be more specific about what the fragment contains. It don't help much as any set of data inside `ProductGeneratingContext` can be a fragment.

Reply:
@msmanoj I've changed it to this: https://github.com/Ensembl/ensembl-client/blob/624548ee763c32519a7050381cda51742c6af291/src/ensembl/src/shared/components/instant-download/instant-download-fetch/fetchSequenceChecksums.ts#L27

Reply:
@imransl Sorry, I meant if the name `ProductGeneratingContextFragment` can be changed to be something more specific.

Reply:
I think it shouldn't have the `ProductGeneratingContext` in its name at all. What we are dealing with here is just checksums.

Reply:
Change it to `SequenceChecksums`  ✅